src/session/sessionReducer.js

```diff
@@ -24,6 +24,16 @@ import { getHasAuth } from '../account/accountsSelectors';
  * See {@link SessionState} for discussion of what "non-persistent" means.
  */
 export type PerAccountSessionState = $ReadOnly<{
+  /**
+   * The event queue ID that we're currently polling on, if any.
+   *
+   * Null when we're not polling on any event queue:
+   * - Between startup and registering a queue
+   * - After the server tells us our old queue was invalid, and before we've
+   *   registered a new one
+   * - While this account is logged out
+   * - While this account is logged in but is not the active account
```

Comment:
I don't think this item applies.

* At present, the only per-account state we have is the one for the active account. So "this account" here is always the active account.
* In a future where we have any per-account session state for accounts other than the active account, I think that will go together with maintaining an event queue for those accounts.